tsdb/query_executor.go

```diff
@@ -248,7 +248,7 @@ func (q *QueryExecutor) plan(stmt *influxql.SelectStatement, chunkSize int) (Exe
 	var executor Executor
 	if len(mappers) > 0 {
 		// All Mapper are of same type, so check first to determine correct Executor type.
-		if _, ok := mappers[0].(*RawMapper); ok {
```

Comment:
Not too happy about this. This check is now in two places -- at the query planner layer, and close to the TSBD store. Works very well, but subject to bugs in the future if one place is updated and the other is not.